spec/support/factories.rb

```diff
@@ -8,19 +13,18 @@
     password "password"
     password_confirmation "password"
 
-      factory :volunteer_with_tasks do
-        transient do
-          tasks_count 2
-        end
-
-        after(:create) do |volunteer, evaluator|
-          create_list(:task, evaluator.tasks_count, volunteer: volunteer)
-        end
-      end
+    # factory :volunteer_with_tasks do
```

Comment:
This is commented out for now as team is deciding whether or not this will be used.